frontend/README.md

```diff
@@ -1,8 +1,30 @@
 ## Available Scripts
 
-In the project directory, you can run:
-### `npm install`
-### `yarn run dev-server`
+This application can be run locally with webpack-dev-server or via a docker container.  Both methods below will result in a web site running in development mode available at http://localhost:8080
 
-Runs the app in the development mode.<br>
-Open [http://localhost:8080](http://localhost:8080) to view it in the browser.
+#### To run locally via webpack-dev-server:
+```
+$> npm install
+$> npm run dev-server
+```
+Visit http://localhost:8080 in your browser.
+
+#### To run locally via Docker container
+Note: Give it some time to start up; it has to install all the dependencies
+```
+docker-compose up -d
+```
+Visit http://localhost:8080 in your browser.
+
+#### To build a production Docker image
+```
+$> docker build --pull --no-cache --force-rm --quiet \
+  -t meep-frontend -f prod.Dockerfile .
+```
+
+#### To run the production image locally
+```
+$> docker run -it -p 8080:80 \
+  $(docker images -f label=project=meep -f label=component=frontend -q)
```

Comment:
Per our slack discussion, I still would push to have a platform agnostic command to run. This could be very helpful for first timers or other contributors who don't know docker that well.

Reply:
I suppose this wouldn't work on Windows. If you know the image ID to run, you can use it.  The command above finds out the ID for you based on the image's labels.